src/js/features/home/home.routes.js

```diff
@@ -1,13 +1,12 @@
-import { IndexRoute, Route } from 'react-router';
-
 import NavbarAndFooterTemplate from '../common/templates/navbarAndFooter/navbarAndFooter.template';
 import HomeComponent from './home.component';
 
-const homeRoutes = (
-    <Route component={NavbarAndFooterTemplate}>
-        {/* Nested route so HomeComponent gets a navbar and footer */}
-        <IndexRoute component={HomeComponent} />
-    </Route>
-);
+const homeRoutes = {
+    path: '',
```

Comment:
I am not sure, but I don't think you need to specify `path` if `path` is an empty string